src/main/java/com/farion/onlinebookstore/dto/item/UpdateCartItemDto.java

```diff
@@ -0,0 +1,12 @@
+package com.farion.onlinebookstore.dto.item;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class UpdateCartItemDto {
+    @NotNull
```

Comment:
Int could not be null, so I believe Min annotation will be enough, or you can make int -> Integer